develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+# Selecting Entities Using the Item Selector API [](id=selecting-entities-using-the-item-selector-api)
```

Comment:
Hey guys, I think I'm not viewing the right tutorial, because this tutorial https://github.com/mwilliams2014/liferay-docs/blob/d462cbc1241ff4c11bb9a33cfe460b2e78d81d15/develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown is actually about how to select entities using the item selector. 

The tutorial that guides you about how to create selectable entities is this one: https://github.com/mwilliams2014/liferay-docs/blob/d462cbc1241ff4c11bb9a33cfe460b2e78d81d15/develop/tutorials/articles/192-item-selector/02-creating-custom-entites-for-the-item-selector.markdown

Let me know if I'm missing something here :)